lib/core.js

```diff
@@ -290,7 +290,7 @@ declare class String {
     anchor(name: string): string;
     charAt(pos: number): string;
     charCodeAt(index: number): number;
-    codePointAt(index: number): number;
+    codePointAt(index: number): ?number;
```

Comment:
`?number` is `void | null | number`. I think `number | void`  would be more accurate.

Reply:
+1, ping me after having fixed this

Reply:
@mrkev have updated this